graviti/operation/frame.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python3
+#
+# Copyright 2022 Graviti. Licensed under MIT License.
+#
+
+"""Definitions of different operations on a DataFrame."""
+
+from typing import Any, Dict, List, Optional
+
+from graviti.openapi import add_data, update_data
+
+
+class DataPost:
```

Comment:
upload data is quite complicated, binary data should be uploaded to oss/s3, also we need multithread to enhance the real time performance. you should consider that.